### ISSUE TITLE
Menu missing: many commands depend on keyboard shortcuts only

### ISSUE THREAD

**Issue by freebrowser1** [Stellarium/stellarium#3827]
### Expected Behavior
There are many options which are hidden and only accessible when clicking the help button (or pressing F1 if the OS or other app does not capture that) and then reading which key I have to press.

<img width="490" alt="Screenshot 2024-07-29 at 17 09 49" src="https://github.com/user-attachments/assets/7ba7d251-d3cf-4f6c-ab40-9c492a7b6345">

But clicking on the action does not work, one has to remember the keyboard shortcut which is a bit from the old MS DOS era. Making this help screen clickable by allowing the user to click on each command and, even better, allow these shortcuts adding to the botton toolbar which pops up when moving the mouse to the bottom of the screen.
That would make Stellarium much more intuitive. People cannot remember such a plethora of keyboard shortcuts.

### System
* Stellarium version: 24.2
* Operating system: Any (macOS and Linux in my case)

**Comment by 10110111** [comment]
Many of them have a reason not to be on the toolbar. E.g. time scrolling is available from the *Time* dialog, checking markers or selecting objects without a click doesn't make sense etc. If you find some shortcuts that aren't available in the GUI but should be, please list them explicitly (but note that GUI is not limited by the toolbar).

**Comment by gzotti** [comment]
How many buttons do you think fit on the screen? Time control can be done in the intuitive time panel for beginners, or using those keyboard shortcuts for power users. The buttons on the toolbar show key shortcuts tooltips on mouse-over. After a few minutes you will know and likely use them if you need the respective functions. 

There are maybe too many options, though. Not everything can be configured into a keyboard with only 105 keys, so you must choose (and then remember) your favourites.

**Comment by 3domennotry** [comment]
Hello.
(Sorry, I can not speak English very well.)

Unfortunately, I think there are two problems.

1.
There is a limit to the width of the screen.
Therefore, additional code is needed to determine if it will be exceeded.

2.
If there are 100 buttons, 100 icons will be needed.

By the way, 
I want hide buttons that are not normally used, but this is not possible.

"Switch between equatorial and azimuthal mount": I have only alt-azimuthal mount.
"Constellation art": It is not included in the sky culture I always use.
"Equatorial grid": I usually use J2000.

Of course, some will need it.

thank you.

**Comment by freebrowser1** [comment]
Indeed, there is a limit to the number of buttons on the button bar because of the screen width.
But at least the F1 screen (screenshot) can be made clickable so that the user can click on each item which is a lot easier than remembering keyboard shortcuts.
